mods/ra/sequences/structures.yaml

```diff
@@ -93,7 +96,7 @@ proc:
 		Length: *
 	dead: procdead
 		Start: 0
-		ZOffset: -1c511
```

Comment:
This is a fundamental limitation of our Effects, which are not z-sorted (and so ignore this property).  We have the same problem with infantry death animations and many other things.

This will eventually be fixable once somebody implements the renderer depth buffer (which is a prerequisite of TS, but will require low-level renderer surgery), but if this is a blocker until then we will need to abandon this PR.